macOS/fastlane/Pluginfile

```diff
@@ -2,4 +2,4 @@
 #
 # Ensure this file is checked in to source control!
 
-gem 'fastlane-plugin-ddg_apple_automation', git: 'https://github.com/duckduckgo/fastlane-plugin-ddg_apple_automation', tag: "2.2.4"
+gem 'fastlane-plugin-ddg_apple_automation', git: 'https://github.com/duckduckgo/fastlane-plugin-ddg_apple_automation', branch: "anh/update_embedded_dbp_broker_jsons"
```

Comment:
After you’ve made a release of the plugin, please update the tag here and also run bundle update to update the lockfile. Please also repeat the same in the iOS subdirectory (there’s a separate pluginfile that needs updating) and in the main directory (no pluginfile there but just bundle update).